concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1338,6 +1338,73 @@ jobs:
                   ./paas-cf/concourse/scripts/set_quotas_from_manifest.rb cf-manifest/cf-manifest.yml
 
       - aggregate:
+        - task: deploy-test-application
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+            params:
+              TEST_HEAVY_LOAD: {{test_heavy_load}}
+            inputs:
+              - name: paas-cf
+              - name: config
+              - name: bosh-CA
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  if [ "${TEST_HEAVY_LOAD:-}" -eq "false" ] ; then
```

Comment:
`-eq` is meant for comparing integers. For strings `=` or `!=` should be used. See `man test` for full details.